test/network/scenarios/p2p/peers.js

```diff
@@ -82,7 +82,7 @@ module.exports = function(configurations, network) {
 						// to update across all nodes.
 						setTimeout(() => {
 							resolve();
-						}, 5000);
+						}, 10000);
```

Comment:
?

Reply:
Network test suite is failing due to the security time to wait for some actions is not enough. I've seen the same problem in propagation scenario.